### ISSUE TITLE
XAML Code Action Split Attributes should allow split to new line

### ISSUE THREAD

**Issue by ravero** [mfractor/mfractor-feedback#180]
This code action is very useful, but it should allow for setting different styles of applying it. The default implementation splits the attributes on lines by keeping the first attribute in the first then aligning the other attributes in the lines below at the same column:

```xml
<Label Text="Welcome to Xamarin.Forms!"
       HorizontalOptions="Center"
       VerticalOptions="CenterAndExpand"/>
```

This is one of the employed standards and is very used in WPF, but another style that is very popular is just by splitting attributes onto its own just adding a tabulation:

```xml
<Label
    Text="Welcome to Xamarin.Forms!"
    HorizontalOptions="Center"
    VerticalOptions="CenterAndExpand" />
```

MFractor would allow applying both styles. This may come as a setting.

**Comment by matthewrdev** [comment]
@ravero We can surface this as a setting in the formatting options panel.

**Comment by ravero** [comment]
This is how the XAML Styler handles that, so I think it makes sense.
Also we can think of adding project specific settings to mfc file.

**Comment by matthewrdev** [comment]
This has been added and will be available in MFractor v4.0.6.

![image](https://user-images.githubusercontent.com/3260473/77123941-3af09600-6a8d-11ea-8c96-05045e30eb91.png)
